### PR TITLE
Config with buildout recipe

### PR DESCRIPTION
**Pull request by icemac**
* Lint the code.
* Add support for Python 3.9.

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -3,11 +3,11 @@ CHANGES
 =======
 
 2.0 (unreleased)
-------------------
+----------------
 
 - Drop support for Python 2.6, 3.2, 3.3.
```
Comment (jugmac00) [inline CHANGES.rst:8]: Maybe somewhen, somebody forget to add 3.4 here?

**Comment by jugmac00** [comment]
Not related to this PR, but maybe a good idea to clean up...

Afaik this is not necessary any more...
```
# namespace package boilerplate
try:
    __import__('pkg_resources').declare_namespace(__name__)
except ImportError:
    from pkgutil import extend_path
    __path__ = extend_path(__path__, __name__)
```

In Python 3 this could go away completely, in Python 2 we have to keep..
```
__import__('pkg_resources').declare_namespace(__name__)
```

File: src/zc/recipe/filestorage/tests.py [hunk 2]
```diff
@@ -12,11 +12,12 @@
 #
 ##############################################################################
 
-import os, re, unittest
-import pkg_resources
```
Comment (jugmac00) [inline src/zc/recipe/filestorage/tests.py:16]: I guess this was removed intentional. It is not used directly, but I am not sure it used some magic or not, but I think its ok to remove it.

**Review by jugmac00 — APPROVED** [review]
LGTM - a couple of inline annotations and comments

**Comment by jugmac00** [comment]
Is buildout.cfg used for anything?

File: CHANGES.rst (hunk 1 above)
Comment (icemac) [reply CHANGES.rst:8]: Probably 3.4 was never supported by this package.

**Comment by icemac** [comment]
> Is buildout.cfg used for anything?

No, so I removed it.

**Comment by icemac** [comment]
Thank you for reviewing this PR. 😃

**Pull request merged**